docs/gno-tooling/cli/gnokey/querying-a-network.md

```diff
@@ -0,0 +1,199 @@
+we can run the following command:
+
+```bash
+gnokey query auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 -remote https://rpc.gno.land:443
+```
+
+With this, we are asking the Portal Loop network to deliver information about the
+specified address. If everything went correctly, we should get the following
+output:
+
+```bash
+height: 0
+data: {
+  "BaseAccount": {
+    "address": "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5",
+    "coins": "227984898927ugnot",
+    "public_key": {
+      "@type": "/tm.PubKeySecp256k1",
+      "value": "A+FhNtsXHjLfSJk1lB8FbiL4mGPjc50Kt81J7EKDnJ2y"
+    },
+    "account_number": "0",
+    "sequence": "12"
+  }
+}
+```
+
+The return data will contain the following fields:
+- `height` - the height at which the query was executed. This is currently not
+  supported and is `0` by default.
+- `data` - contains the result of the query.
+
+The `data` field returns a `BaseAccount`, which is the main struct used in [TM2](../../../concepts/tendermint2.md)
+to hold account data. It contains the following information:
+- `address` - the address of the account
+- `coins` - the list of coins the account owns
+- `public_key` - the TM2 public key of the account, from which the address is derived
+- `account_number` - a unique identifier for the account on the gno.land chain
+- `sequence` - a nonce, used for protection against replay attacks
+
+## `bank/balances`
+
+With this query, we can fetch [coin](../../../concepts/stdlibs/coin.md) balances
+of a specific account. To call it, we can run the following command:
+
+```bash
+gnokey query bank/balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 -remote https://rpc.gno.land:443
+```
+
+If everything went correctly, we should get an output similar to the following:
+
+```bash
+height: 0
+data: "227984898927ugnot"
+```
+
+The data field will contain the coins the address owns.
+
+## `vm/qfuncs`
+
+Using the `vm/qfuncs` query, we can fetch exported functions from a specific package
+path. To specify the path we want to query, we can use the `-data` flag:
+
+```bash
+gnokey query vm/qfuncs --data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the `wugnot` realm:
+
+```json
+height: 0
+data: [
+        {
+          "FuncName": "Deposit",
+          "Params": null,
+          "Results": null
+        },
+        {
+          "FuncName": "Withdraw",
+          "Params": [
+            {
+            "Name": "amount",
+            "Type": "uint64",
+            "Value": ""
+            }
+          ],
+          "Results": null
+        },
+        // other functions
+]
+```
+
+## `vm/qfile`
+
+With the `vm/qfile` query, we can fetch files found on a specific package path.
+To specify the path we want to query, we can use the `-data` flag:
+
+```bash
+gnokey query vm/qfile -data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the
```

Comment:
I think you meant to say the output is the list of files in the wugnot package.

Reply:
Good catch, thanks: [f96159f](https://github.com/gnolang/gno/pull/2226/commits/f96159f9d86508d1448bdf831291da099ae5ff36)